clients/http/schedulejob.go

```diff
@@ -70,9 +71,23 @@ func (client ScheduleJobClient) AllScheduleJobs(ctx context.Context, offset int,
 // ScheduleJobByName queries the schedule job by name
 func (client ScheduleJobClient) ScheduleJobByName(ctx context.Context, name string) (
 	res responses.ScheduleJobResponse, err errors.EdgeX) {
-	path := common.NewPathBuilder().EnableNameFieldEscape(client.enableNameFieldEscape).
+	requestPath := common.NewPathBuilder().EnableNameFieldEscape(client.enableNameFieldEscape).
 		SetPath(common.ApiScheduleJobRoute).SetPath(common.Name).SetNameFieldPath(name).BuildPath()
-	err = utils.GetRequest(ctx, &res, client.baseUrl, path, nil, client.authInjector)
+	err = utils.GetRequest(ctx, &res, client.baseUrl, requestPath, nil, client.authInjector)
+	if err != nil {
+		return res, errors.NewCommonEdgeXWrapper(err)
+	}
+	return res, nil
+}
+
+// ScheduleJobsByLabel queries the schedule jobs by label
+func (client ScheduleJobClient) ScheduleJobsByLabel(ctx context.Context, label string, offset, limit int) (
```

Comment:
just add a parameter to AllScheduleJobs, and the labels should be a slice

example:
https://github.com/edgexfoundry/go-mod-core-contracts/blob/f0e7cf591de664f5b10473d12b78eb269826aa13/clients/http/device.go#L80

Reply:
Done.